pkg/config/model/ipc_address.go

```diff
@@ -0,0 +1,44 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package model
+
+import (
+	"fmt"
+	"net"
+)
+
+// IsLocalAddress returns the given address if it is local or an error if it is not
+func IsLocalAddress(address string) (string, error) {
```

Comment:
This isn't part of `model`. It might better fit under `pkg/config/env`

Reply:
+1 to this suggestion

only comment I have is this feels out of place on the model and should be part of the `pkg/config/env` or moved to something like `pkg/config/utils/ip_address`